Makefile

```diff
@@ -146,6 +156,13 @@ rm -rf $$TMP_DIR ;\
 }
 endef
 
+DEPLOYMENT_DIR = $(PROJECT_DIR)/config/deploy
+.PHONY: deploy-manifest
+deploy-manifest:
+	mkdir -p $(DEPLOYMENT_DIR)
+	cd $(PROJECT_DIR)/config/manager && $(KUSTOMIZE) edit set image controller=$(IMG) ;\
```

Comment:
This will leave an uncommitted change. You may want to rollback to the default `quay.io/kuadrant/limitador-operator:latest` after building the manifests.

Reply:
Ok, added a cleanup cmd.